infrastructure/locals.tf

```diff
@@ -0,0 +1,4 @@
+locals {
+    bucketname-upload = "nfish-des-kutter-upload"
+    bucketname-store = "nfish-des-kutter-store"
+}
```

Comment:
if you use a variable and the bucket name as default you can also configure it. Maybe for a test environment to use: 

nfish-des-kutter-upload-dev